src/frontend/Komponenter/Behandling/VedtakOgBeregning/Felles/SelectVedtaksresultat.tsx

```diff
@@ -55,7 +56,8 @@ const SelectVedtaksresultat = (props: Props): JSX.Element => {
 
     const { settVisNullstillVedtakModal } = useContext(NullstillVedtakModalContext);
 
-    const skalViseNullstillKnapp = toggles[ToggleName.visNullstillVedtakKnapp];
+    const skalViseNullstillKnapp =
+        behandling.steg === Steg.SEND_TIL_BESLUTTER && toggles[ToggleName.visNullstillVedtakKnapp];
```

Comment:
Saksbeh. går tilbake endrer vilkår. Da er vi vel ikke i SEND_TIL_BESLUTTER lenger.

Reply:
Blir ikke dette riktig for det? Man vil vel ikke vise knappen for nullstilling av vedtak dersom man er i `VILKÅR` fasen?

Reply:
Det kan være et vedtak man vil nullstille selv om man er i steget VILKÅR, så derfor blir det som Mattis nevner ikke helt riktig.

Vi snakket litt om denne på fredag og konklusjonen ble at vi ikke kan basere oss på steget, men heller sjekke manuelt om det er et lagret vedtak som kan slettes.